modules/lite/src/index.js

```diff
@@ -7,12 +7,13 @@
 const lumaGL = require('luma.gl');
 const deckGLCore = require('@deck.gl/core');
 const deckGLCoreLayers = require('@deck.gl/layers');
+const DeckLayer = require('@deck.gl/mapbox-layers').DeckLayer;
```

Comment:
No impact on this PR, just heads up for audit:

I am not happy with the `DeckLayer` name, but haven't been able to think of anything better. It can make sense to import it as `DeckLayer` but it seems weird to export it with this name.

Reply:
`MapboxLayer` is probably more appropriate in this context.

Speaking of which, should we rename the module `mapbox-layer` without an s?
Or a more future-proof `mapbox-plugins`?

Reply:
> Speaking of which, should we rename the module mapbox-layer without an s?
Or a more future-proof mapbox-plugins?

Yes...

The current placeholder module name came from the idea that we would present the new module as a "deck.gl layer pack" for mapbox. But since the layers are not included, that is not really true, so we can change it.

The most future proof name would probably just be `@deck.gl/mapbox`. Anything mapbox dependent would go here. `@deck.gl/mapbox-utils` or `@deck.gl/mapbox-tools` are other possibilities.


App could would be something like...
```
import {MapboxLayer as DeckMapboxLayer} from '@deck.gl/mapbox';
import {...} from '@deck.gl/layers';
```

I think it works.